collections/_sdk/effects/appointment-labels.md

```diff
@@ -0,0 +1,152 @@
+| `appointment_id` | `str`     | ID of the appointment to add labels to                                  | Yes      |
+| `labels`         | `set[str]`| Set of label names to add (1-3 labels total per appointment)            | Yes      |
+
+### apply() → Effect
+
+Adds the specified labels to the appointment.
+
+#### Returns
+
+An `Effect` object configured for adding appointment labels.
+
+#### Behavior
+
+- Labels are added to the appointment if the total count doesn't exceed 3
+- Labels are automatically sorted for consistency
+- Duplicate labels are ignored (labels are stored as a set)
+- Validates the appointment exists before adding labels
+- Validates label names are non-empty strings
+- Returns an error if adding labels would exceed the 3-label limit
+
+#### Example Usage
+
+```python?partial=true
+from canvas_sdk.effects.note.appointment import AddAppointmentLabel
+from canvas_sdk.handlers.base import BaseHandler
+
+class MyProtocol(BaseHandler):
+    def compute(self):
+        # Add labels to an appointment
+        effect = AddAppointmentLabel(
+            appointment_id="appointment-uuid",
+            labels={"URGENT", "FOLLOW_UP"}
+        )
+
+        return [effect.apply()]
+```
+
+#### Error Handling
+
+```python?partial=true
+from canvas_sdk.effects.note.appointment import AddAppointmentLabel
+from canvas_sdk.exceptions import ValidationError
+
+def handle_validation_errors():
+    # Example of handling validation errors
+    try:
+        effect = AddAppointmentLabel(
+            appointment_id="invalid-id",
+            labels={"LABEL1", "LABEL2", "LABEL3", "LABEL4"}  # Would exceed limit
+        )
+        return [effect.apply()]
+    except ValidationError as e:
+        # Handle validation errors
+        return []
+```
+
+---
+
+## RemoveAppointmentLabel Effect
+
+The `RemoveAppointmentLabel` effect removes one or more labels from an existing appointment.
+
+### Attributes
+
+| Attribute        | Type      | Description                                                             | Required |
+|------------------|-----------|-------------------------------------------------------------------------|----------|
+| `appointment_id` | `str`     | ID of the appointment to remove labels from                             | Yes      |
+| `labels`         | `set[str]`| Set of label names to remove                                            | Yes      |
+
+### apply() → Effect
+
+Removes the specified labels from the appointment.
+
+#### Returns
+
+An `Effect` object configured for removing appointment labels.
+
+#### Behavior
+
+- Removes the specified labels from the appointment
+- Non-existent labels are ignored (no error thrown)
+- Validates the appointment exists before removing labels
+
+#### Example Usage
+
+```python?partial=true
+from canvas_sdk.effects.note.appointment import RemoveAppointmentLabel
+from canvas_sdk.handlers.base import BaseHandler
+
+class MyProtocol(BaseHandler):
```

Comment:
Call it `MyHandler` instead and provide a `RESPONDS_TO` for the class to make it a functional handler.